configure.ac

```diff
@@ -34,6 +34,20 @@ AC_SUBST(ODPH_VERSION_MAJOR)
 ODPH_VERSION_MINOR=odph_version_minor
 AC_SUBST(ODPH_VERSION_MINOR)
 
+##########################################################################
+# Use these default flags if user has not defined CFLAGS. Otherwise,
+# automake will initialize CFLAGS to "-g O2", and it would not be possible
+# to check if CFLAGS are user defined.
+##########################################################################
+: ${CFLAGS=""}
+if test -z "$CFLAGS"; then
+	user_cflags=0
+	CFLAGS="-O2 -g"
+else
+	user_cflags=1
+fi
```

Comment:
`AS_IF([test "$ac_cv_env_CFLAGS_set" = ""], [...], [...])`

Reply:
Fixed and added your review tag in v4. Thanks.

Reply:
Thank you.